examples/gateway/azure_openai/README.md

```diff
@@ -0,0 +1,63 @@
+## Example route configuration for AzureOpenAI
+
+The following example configuration shows the 3 supported routes for AzureOpenAI: chat, completions, and embeddings.
+Additionally, it illustrates the two separate api types that are supported for this service.
+
+- _"azure"_ api type: uses a generated token that is applied by setting the API token key directly to an environment variable
+- _"azuread"_ api type: uses Azure Active Directory for supplying the active directory key to be used to an environment variable
```

Comment:
```suggestion
- `azure` api type: uses a generated token that is applied by setting the API token key directly to an environment variable
- `azuread` api type: uses Azure Active Directory for supplying the active directory key to be used to an environment variable
```

Reply:
I like the code formatting better :) thanks